uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -285,11 +285,22 @@ function tasksAssignments_civicrm_alterAngular(\Civi\Angular\Manager $angular) {
     _tasksAssignments_change_workflow_help_text($doc);
     _tasksAssignments_remove_non_civihr_tabs_from_workflow($doc);
     _tasksAssignments_allow_only_add_timeline_action($doc);
+
+    $doc->find('.crmCaseType')
+    ->attr('ng-controller', 'caseTypeExtendedController');
   });
 
   $angular->add($changeSet);
 }
 
+function tasksAssignments_civicrm_angularModules(&$angularModules) {
+  $angularModules['crmTaskWorkflows'] = array(
+    'ext' => 'civicrm',
+    'js' => array('tools/extensions/civihr_tasks/uk.co.compucorp.civicrm.tasksassignments/js/dist/crm-task-workflows.min.js'),
```

Comment:
I went for defining a new module `crmTaskWorkflows` which will contain all overridden module. Because other we will need to define each module separately

```php
function tasksAssignments_civicrm_angularModules(&$angularModules) {
  $angularModules['crmCaseTypeExtended'] = array(
    'ext' => 'civicrm',
    'js' => array(''),
    'requires' => array('crmCaseType'),
  );
  $angularModules['someThingElseExtended'] = array(
    'ext' => 'civicrm',
    'js' => array(''),
    'requires' => array('crmCaseType'),
  );
  ....
}